iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -637,16 +638,22 @@ public MemberExpression MEMBER_EXPRESSION_FUNCTION_CALL_COMPONENT() {
   public MemberExpression MEMBER_EXPRESSION_IDENTIFIER_COMPONENT() {
     return b.<MemberExpression>nonterminal().is(
       f.memberExpressionComponent(
-        b.firstOf(
-          b.token(Punctuator.DOT),
-          b.token(Punctuator.DOUBLE_COLON)),
+        f.tuple(b.token(Punctuator.DOT), b.optional(b.token(Punctuator.SAFE_DEREFERENCE))),
```

Comment:
I think you can make this a function with 3 parameters instead of passing a tuple